### DISCUSSION TITLE
Long text on mobile desfigures site

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#233] [General]
On menus descriptions if we use text longer than width it desfigures the site.

![pushevs-text-outside-the-box](https://github.com/hbstack/theme-cards/assets/49299328/c68972a6-8fdb-4479-a5ca-9a68254e4151)

Not very serious, I can just reduce the menu description.

The menu in question is [Stores](https://pushevs.com/).

**Comment by razonyang** [comment]
Closed by hbstack/header#558
